ui/js/services/main.js

```diff
@@ -20,12 +20,7 @@ treeherder.factory('thUrl', [
                 return thUrl.getProjectUrl(uri, repoName);
             },
             getJobsUrl: function (repo, fromChange, toChange) {
-                return "index.html#/jobs?" + _.reduce({
-                    repo: repo, fromchange: fromChange, tochange: toChange
-                }, function (result, v, k) {
-                    if (result.length) result += '&';
-                    return result + k + '=' + v;
-                }, "");
+                return "index.html#/jobs?repo=" + repo + "&fromchange=" + fromChange + "&tochange=" + toChange;
```

Comment:
Could you switch this to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) - I think it will read more clearly :-)